### PR TITLE
Add pausing membership blog post

### PR DESCRIPTION
**Pull request by jnewbery**

### CODE REVIEW

**Comment by jnewbery** [comment]
For release on 2021-02-19 if everyone is happy. I've addressed all the review comments from https://github.com/bitcoinops/bitcoinops.github.io/commit/e8526ffc116d4be1f15aae2a7ef5b2f3dafe8f75.

File: _posts/en/2021-02-19-pausing-memberships.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+We [launched Bitcoin Optech][Optech launch] in July 2018 with a very clear
+mission: to help Bitcoin companies adopt scaling technologies and to raise
+the level of conversation around scaling Bitcoin. Over the previous months,
+the Bitcoin community had been through the bitter and contentious 'fork wars',
+and our ethos was (and remains) to focus on the technology and provide
+sound technical guidance for anyone wishing to use the Bitcoin block chain in
+a more efficient way.
+
+Our work is made possible by very generous founding donations from John
+Pfeffer, Wences Casares and Chaincode Labs, and ongoing funding from our
```
Comment (harding) [inline _posts/en/2021-02-19-pausing-memberships.md:31]: Nit: suggest changing the second "and" to "with".

**Review by harding — APPROVED** [review]
Untested ACK with one tiny suggestion.

File: _posts/en/2021-02-19-pausing-memberships.md (hunk 1 above)
Comment (jnewbery) [reply _posts/en/2021-02-19-pausing-memberships.md]: I agree that's better. Thanks!

**Comment by jnewbery** [comment]
Thanks for the review @adamjonas! I've taken your suggestion.

**Review by bitschmidty — APPROVED** [review]
tACK f7aac04aeb22bb68defb40c264656aaed07e9ce0

**Comment by jnewbery** [comment]
Thanks Mike!

**Pull request merged**